src/appkit.cs

```diff
@@ -13069,6 +13143,30 @@ partial interface NSScreen {
 		string LocalizedName { get; }
 	}
 
+	[NoMacCatalyst]
+	partial interface NSScreen
```

Comment:
Was unsure about what to do about this interface structure - there are a couple other partial interfaces with the same name, in the same file here. Not sure why they're all separated, but sharpie does the same, and diffs suggest the interface is new for some reason. So, I just did what the diffs/sharpie seem to indicate.

But I was wondering - why is it like that? Can't I just add these new properties to the partial interface above?

Reply:
I don't know why sharpie does that, but you can just add it to any existing partial interfaces (at the very least it makes it less confusing for the next person).

Reply:
sounds good, thanks!